avrocat/consumer.py

```diff
@@ -33,7 +34,7 @@ def __init__(self, **kwargs):
             'group.id': self._group,
             'enable.auto.commit': True,
             'default.topic.config': {
-                'auto.offset.reset': 'earliest'
+                'auto.offset.reset': self._offset_reset or 'earliest'
```

Comment:
You don't have to specify a default value here since you define a "[default: earliest]" in the docstring (handled by docopt)

Reply:
Oops :). Fixed.